modules/opp-architecture-installing-policyset.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_module-type: PROCEDURE
+[id="opp-architecture-installing-policyset_{context}"]
+= Installing {product-title} by using PolicySets
+
+The {ocp} installation uses two `PolicySets` to install additional products.
+
+[NOTE]
+====
+Edit the `policyGenerator.yaml` file to remove any product that you do not want to install. You can delete the product entry or add comment characters on the line.
+====
+
+.Procedure
+. Install the `PolicyGenerator` plug-in by using _Installing and using the PolicyGenerator Kustomize plug-in_.
```

Comment:
After installing the PolicyGenerator, we need this step too:
- Clone the policy-collection repository: `git clone https://github.com/stolostron/policy-collection`
- Go to the directory for the `PolicySet`: `cd policygenerator/policy-sets/stable/openshift-plus`